source/adios2/engine/bp4/BP4Writer.cpp

```diff
@@ -20,6 +20,8 @@
 #include <ctime>
 #include <iostream>
 
+#include "scr.h"
```

Comment:
Yes, good catch.

Reply:
I added this guard in the BP4 engine.  I've left them out of the examples for now.  It may be better to create SCR-specific examples so as not to clutter the hello examples.  If we are to merge this, I can make that change before the merge.